apps/web/src/utils/nameUtils.tsx

```diff
@@ -118,7 +118,7 @@ export const getFullNameHtml = (
  * @return string
  */
 export const splitAndJoin = (text: string, split?: string, join?: string) =>
-  text.split(split || "").join(join || " ");
+  text.split(split ?? "").join(join ?? " ");
```

Comment:
This function might be easier to read using parameters with default values instead of coalescing.

Reply:
Accidentally added this into Updated in [fix localized names null fallbacks](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11651/commits/37d38ad80f04bf3718d0ba3b6d072894c2fe03c1)